pull-request.py

```diff
@@ -283,7 +282,9 @@ def add_reviewers(entry, reviewers, team_reviewers):
 ################################################################################
 
 API_VERSION = "v3"
-BASE = "https://api.github.com"
+
+# Allow for a GitHub enterprise URL
+BASE = get_envar("PULL_REQUEST_API_URL") or "https://api.github.com"
```

Comment:
GitHub Actions sets automatically the base API url as the [`GITHUB_API_URL`](https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables) environment variable. We could use that and this will be either `https://api.github.com` or `https://github.enterprise.com/api/v3` for GitHub Enterprise Server.

```suggestion
BASE = get_envar("GITHUB_API_URL")
```

Reply:
oh yes, that would be much better!